modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/home/app.scss

```diff
@@ -0,0 +1,3 @@
+@import '../../../src/common/styles/global';
+@import 'Home';
```

Comment:
Hi @wandersonvicente, it's better merge app.scss with Home.scss because both are about the same component/route, the Home.

Reply:
Merged home into app. Done.